style.css

```diff
@@ -0,0 +1,360 @@
+}
+
+.works-title h1 {
+  margin: 5px auto;
+}
+
+.work-container {
+  display: grid;
+  width: 331px;
+  margin-top: 50px;
+  gap: 25px;
+}
+
+.cart-container {
+  width: 100%;
+  height: 474px;
+  border-radius: 8px;
+  border: 1px solid #d0d9d4;
+  background-color: #ebf0ee;
+}
+
+.img-container {
+  width: 100%;
+  height: 252px;
+}
+
+.cart-datails {
+  width: calc(100% - 22px);
+  height: 200px;
+  padding: 11px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 1rem;
+  background: #fff;
+  border-radius: 8px;
+}
+
+.project-title {
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  margin: 5px;
+  display: flex;
+  text-align: center;
+  color: #3a4a42;
+}
+
+.project-links {
+  display: flex;
+  align-items: stretch;
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+  gap: 6px;
+  height: 32px;
+}
+
+.project-links li {
+  flex: 1;
+  text-align: center;
+  padding: 8px 5px;
+  background: #ebf0ee;
+  border-radius: 4px;
+  font-size: 12px;
+  font-weight: 600;
+  line-height: 16px;
+}
+
+.project-links li a {
+  text-decoration: none;
+  color: #3a4a42;
+  white-space: nowrap;
+}
+
+.cart-datails .btn {
+  height: 20px;
+  font-weight: 500;
+  cursor: pointer;
+}
+
+/* Style for about me section */
+.about-section {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin: 0 auto;
+  justify-items: center;
+  align-items: center;
+}
+
+.about-header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  background: url("images/aboutme-topright.svg"), url("images/aboutme-bottomleft.svg");
+  background-position: right top, left bottom;
+  background-repeat: no-repeat;
+}
+
+.about-header h1 {
+  width: 60%;
+  font-size: 1.8em;
+  font-weight: 700;
+  text-align: center;
+}
+
+.about-header p {
+  width: 70%;
+  margin: 0 auto 25px;
+  line-height: 1.6;
+  font-weight: 500;
+  text-align: center;
+}
+
+.about-header .btn {
+  width: 135px;
+  font-weight: 500;
+  text-align: center;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.about-container {
+  display: grid;
+  width: 331px;
+  margin-top: 50px;
+  gap: 25px;
+  align-self: center;
+}
+
+.about-cart {
+  width: 100%;
+  height: 316px;
+  padding-bottom: 5%;
+  border-radius: 8px;
+  border: 1px solid #c1c7d0;
+  background-color: #ebf0ee;
+}
+
+.about-icon {
+  width: 100%;
+  height: 100px;
+  margin-top: 30px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.about-datails {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.about-datails h2 {
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  text-align: center;
+  color: #344563;
+}
+
+.language-li {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  list-style-type: none;
+  padding: 0;
+  height: 40px;
+  gap: 8px;
+}
```

Comment:
- [ ] Kindly add some extra `space/margins/padding` around the `list` items to match the given design.

- Your Design:
- ![image](https://user-images.githubusercontent.com/60083437/227313784-43b3c64b-ff21-4d48-abfa-a1e1cad4e21b.png)

Reply:
Dear @Tufahel, I solved this problem add some margins to the ul lists.